lib/rust/parser/src/metadata.rs

```diff
@@ -0,0 +1,74 @@
+//! Data associated with a syntax tree.
```

Comment:
These docs do not really explain what this module does. They shoudl explain thwt this is about metadata on the bottom of the file with such and such section header.